doc/edge-metrics-v2.md

```diff
@@ -0,0 +1,43 @@
+# IoT Edge Metrics
+
+## IoT Edge Hub
```

Comment:
Should we have separate docs for EdgeHub metrics and EdgeAgent metrics, etc?

Reply:
One place might be good, at least while in experimental state.